.changeset/curvy-cups-cough.md

```diff
@@ -0,0 +1,5 @@
+---
+'svelte': patch
+---
+
+feat: snippets can now take multiple arguments
```

Comment:
I marked this as breaking because the type signature has changed. Maybe we can combine the two changesets into one like this:
```suggestion
breaking: snippets can now take multiple arguments, support default parameters. Because of this, the type signature has changed
```